### PR TITLE
Fix TimeDifference property values

### LINKED ISSUE #4452: Bug: results on getX() not equal
At the doc: https://codeigniter4.github.io/userguide/libraries/time.html#viewing-differences
image

**Describe the bug**
The result is not equal,
one is positive value
and the other is negative value

```PHP
echo $diff->getSeconds();
// return: -220924800

echo $diff->seconds;
// return: 220924800
```

**CodeIgniter 4 version**
v4.1.1

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- Fixes TimeDifference property values
- Fixes #4452
- Fixes user guide classname notation
- Fix time zone bug in TimeDifference https://github.com/codeigniter4/CodeIgniter4/pull/4453/commits/dfba12806ff658ba7bc6956a1d609754626048ed

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPdocs
- [x] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

**Comment by kenjis** [comment]
```
There was 1 failure:

1) CodeIgniter\I18n\TimeDifferenceTest::testGetter
Failed asserting that 7 matches expected -8.

/home/runner/work/CodeIgniter4/CodeIgniter4/tests/system/I18n/TimeDifferenceTest.php:210
```

File: tests/system/I18n/TimeDifferenceTest.php [hunk 1]
```diff
@@ -194,12 +202,25 @@ public function testHumanizeNoDifference()
...
+
+	public function testGetterChicagoTime()
 	{
 		$current = Time::parse('March 10, 2017', 'America/Chicago');
 		$diff    = $current->difference('March 18, 2017', 'America/Chicago');
 
-		$this->assertEquals(-8, (int) round($diff->days));
+		$this->assertEquals(7, $diff->getDays());
+		$this->assertEquals(7, $diff->days);
+		$this->assertEquals(-8, (int) round($diff->getDays(true)));
```
Comment (kenjis) [inline tests/system/I18n/TimeDifferenceTest.php:223]: I don't know why the difference is 7 days in Chicago time.

Comment (kenjis) [reply tests/system/I18n/TimeDifferenceTest.php]: It is probably because of summer time.

Comment (iRedds) [reply tests/system/I18n/TimeDifferenceTest.php]: For example, DateTime::diff() does not have this problem.

**Review by MGatner — APPROVED** [review]

**Review by paulbalandan — APPROVED** [review]

**Pull request merged**